src/pages/admin/newportal/contenulibre/index.tsx

```diff
@@ -27,30 +19,18 @@ const ContenuLibre = () => {
     );
   };
 
-  const onNext = () => {
-    const success = createNewActivity(ActivityType.CONTENU_LIBRE, selectedPhase);
-    if (success) {
-      router.push('/admin/newportal/contenulibre/1');
-    }
-  };
-
   if (!isModerator) {
     return <h1>Vous n&apos;avez pas accès à cette page, vous devez être modérateur.</h1>;
   }
 
   return (
-    <Base hideLeftNav>
-      <div style={{ width: '100%', padding: '0.5rem 1rem 1rem 1rem' }}>
-        <div className="width-900">
-          {backButton()}
-          <p className="text">
-            Dans cette activité, nous vous proposons de créer une publication libre. Vous pourrez ensuite partager cette publication et décider de
-            l&apos;épingler dans le fil d&apos;actualité.
-          </p>
-          <StepsButton next={onNext} />
-        </div>
+    <div>
+      <div>
+        {backButton()}
```

Comment:
**Suggestion:** (Not from this PR but I see it here)
Avoid functions returning JSX, it's not the best practice to do with React.

**Without args**
The `backButton` function is created on each render and does not need any argument so it's not needed. Let's directly create a variable storing a component:
```jsx
const backButton = <a>{...}</a>;
[...]
return <div>{backButton}</div>; 
```

**With args**
If the function takes args, I prefer to see it outside the component as another component. This will improve performance with React.
```jsx
const BackButton = (backButtonProps) => <a>[...]</a>;

const ContenuLibre = () => {
    [....]
    return <div><BackButton /></div>;
}
```